packages/bpk-component-banner-alert/src/BpkBannerAlert-test.js

```diff
@@ -55,7 +55,11 @@ describe('BpkBannerAlert', () => {
 
```

Comment:
We should get a test for the new `bannerClassName` prop

Reply:
tru dat!